spdk-sys/build.rs

```diff
@@ -98,7 +98,6 @@ fn main() {
         .derive_default(true)
         .derive_debug(true)
         .derive_copy(true)
-        .clang_arg("-march=nehalem")
```

Comment:
Quick question: should we still specify target architecture based on the desired architecture (ARM, x86_64) or it can be completely omitted and clang does all the required magic without any hints here ?

Reply:
I am fairly certain we'll need to do architecture level defintions here. :(